### ISSUE TITLE
when reporting racks by workspace datacenter is empty

### ISSUE THREAD

**Issue by lderezinski** [joyent/conch#322]
When reporting racks by workspaces, the datacenter field is "" I tried against all SPC build  workspaces with the same results

```script
 ./bin/conch -j ws $i  racks|json -ag
```
```json
{
  "id": "47caeaf6-9393-4d29-8cde-a14a35762077",
  "name": "3.1-0905",
  "role": "MANTA",
  "unit": 0,
  "size": 45,
  "datacenter": "",
  "slots": null
}
```

**Comment by lderezinski** [comment]
also on current version, built locally 

> $ ./bin/conch --version                                                                                                                      [±master ●]
1.3.1
                                                                                                                                                                   
linda@LSD-j2262 ~/go/src/github.com/joyent/conch-shell                                                                                                  [17:35:45] 
> $ git status                                                                                                                                 [±master ●]
On branch master
Your branch is up to date with 'origin/master'.

**Comment by karenetheridge** [comment]
@sungo sungo: the /workspace/:id/relay endpoint returns a list of hashes with the keys: id, alias, created, ipaddr, ssh_port,
updated, version, devices, location.  There is no datacenter field here. Is
the shell turning 'location' into 'datacenter'?

**Comment by sungo** [comment]
This is a bug in the conch shell, tracked under joyent/conch-shell#127 and fixed in joyent/conch-shell#128. The next release of the shell will hopefully be in the latter half of the week.
